config/settings/base.py

```diff
@@ -352,6 +352,9 @@
         "user": "ami.users.api.serializers.UserSerializer",
         "current_user": "ami.users.api.serializers.CurrentUserSerializer",
     },
+    "PERMISSIONS": {
+        "user_create": ["rest_framework.permissions.IsAdminUser"],
```

Comment:
Let me know if you would do it like this @mihow? I had a look here: https://djoser.readthedocs.io/en/latest/settings.html

Reply:
Thank you for finding this setting. Admin for user_create is perfect for the demo. I will make it environment-specific next week.